frontend/assets/user_defined_in_basic.js

```diff
@@ -0,0 +1,42 @@
+function UserDefinedInBasic() {
+}
+
+UserDefinedInBasic.prototype.init = function(record_type, fields, read_only_view) {
+    var bi = $("#basic_information");
+
+    if (bi.length == 0) {
```

Comment:
At first I didn't know why you'd done this, but now I realise the resource and digital object pages fire the initialize events more than once, and some of them are before the form is displayed.  Yay.